code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -812,10 +824,14 @@
 	kronkaine_fiend.set_jitter_if_lower(20 SECONDS * REM * seconds_per_tick)
 	kronkaine_fiend.AdjustSleeping(-2 SECONDS * REM * seconds_per_tick)
 	kronkaine_fiend.adjust_drowsiness(-10 SECONDS * REM * seconds_per_tick)
-	if(volume < 10)
+	// Do not try to cheese the overdose threshhold with purging chems to become stamina immune, if you purge and take stamina damage you will be punished!
+	if(kronkaine_fiend.getStaminaLoss() < 30)
 		return
 	for(var/possible_purger in kronkaine_fiend.reagents.reagent_list)
 		if(istype(possible_purger, /datum/reagent/medicine/c2/multiver) || istype(possible_purger, /datum/reagent/medicine/haloperidol))
+			if(kronkaine_fiend.HasDisease(/datum/disease/adrenal_crisis))
+				break
+			kronkaine_fiend.visible_message(span_bolddanger("[kronkaine_fiend.name] suddenly tenses up, it looks like the shock is causing their body to shut down!"), span_userdanger("You feel your adrenaline peak and then suddenly crash! Maybe you shouldn't have mixed kronkaine and purgatives..."))
```

Comment:
I feel negative about placing this information here of all places. Image getting told X plus Y is bad only after it happens, furthermore with the assumption that you consciously mixed the two. Shouldn't you have this warning somewhere more logical, like the reagent description?

Reply:
I can add a line to the reagent description if you wish

But, players don't read the reagent description though, right? 
Where is it even available, in the chem master?

If people want perfect knowledge and don't want to rely on transmitted peer knowledge, I’d assume they generally consult the wiki.

Reply:
On the actual drug item maybe?

Reply:
I've added a warning to the actual drug item. 

I will update the wiki as well.